sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/cryptography/CryptographyClientTest.java

```diff
@@ -204,6 +244,70 @@ public void signVerifyEc(HttpClient httpClient, CryptographyServiceVersion servi
                 assertTrue(verifyStatus);
             }
         }
+    }
+
+    @Test
+    public void signVerifyEc() throws NoSuchAlgorithmException, InvalidAlgorithmParameterException {
```

Comment:
Good to test these in combinations of sign local, verify remote + sign remote, verify local. Same for encryption and decryption. Of course, for async keys you'd have to create it locally then import to have the private key material. I wouldn't hold up this PR for it, but you might at least add one or two just to validate any local crypto support you have aligns with the service.